sdk/storage/Azure.Storage.Files.DataLake/src/Azure.Storage.Files.DataLake.csproj

```diff
@@ -10,7 +10,7 @@
     <DefineConstants>DataLakeSDK;$(DefineConstants)</DefineConstants>
     <PackageTags>Microsoft Azure Storage Files;Microsoft;Azure;File;Files;Data Lake;Storage;StorageScalable;$(PackageCommonTags)</PackageTags>
     <Description>
-      This client library enables working with the Microsoft Azure Storage File service for storing binary and text data.
+      This client library enables working with the Microsoft Azure Storage Data Lake File service for storing binary and text data.
```

Comment:
I searched DataLake through nuget on VS and we weren't the first ones to show up cause we didn't have Data Lake in the description. Just added this so the search algorithm hopefully picks us up first